TeslaSUI/TeslaSUI/Views/View/CustomSliderView.swift

```diff
@@ -0,0 +1,89 @@
+// CustomSliderView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Кастомный слайдер зарядки
+struct CustomSliderView: View {
+    // MARK: - Public Properties
+
+    @Binding var sliderValue: Double
+
+    var body: some View {
```

Comment:
```suggestion
    var body: some View {
```
очень много всего в боди, включая расчеты и тд
кажется лучше все разнести

Reply:
я пробовал разнести все это, но т.к используется GeomtryReader где задаются размеры которые используются в вью, с выносом в метод который принимает параметр у меня тоже не получается...